presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -559,6 +559,10 @@ Procedures
 
     Create an empty partition in the specified table.
 
+* ``system.msck(schema_name, table_name, add, drop)``
```

Comment:
- I am not sure if we should follow the hive naming. 
- what are `add` and `drop`? booleans? Maybe we could have separate methods to repair separate things?